agir/groups/components/groupPage/GroupPage/Routes/InfoRoute.js

```diff
@@ -48,8 +48,9 @@ const MobileInfoRoute = (props) => {
       {group && group.hasMessages ? (
         <MessagesRoutePreview {...props} />
       ) : group.isManager ? (
-        <EmptyMessages goToMessages={goToMessagesTab} />
+        <PromoMessage goToMessages={true} onClick={goToMessagesTab} />
```

Comment:
```suggestion
        <PromoMessage goToMessages onClick={goToMessagesTab} />
```
Pas bloquant, mais si c'est `true` tu peux ne pas mettre de valeur

Reply:
En effet, c'est chose faite !